v2/broker.go

```diff
@@ -294,6 +294,23 @@ func (b *Broker) CreateTopic(topics []proto.TopicInfo, timeout time.Duration, va
 	return resp, err
 }
 
+// DeleteTopic request topic delition
+func (b *Broker) DeleteTopic(topics []string, timeout time.Duration) (*proto.DeleteTopicsResp, error) {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	var resp *proto.DeleteTopicsResp
+	err := b.callOnClusterController(func(c *connection) error {
+		var err error
+		req := proto.DeleteTopicsReq{
+			Topics:  topics,
+			Timeout: timeout,
+		}
+		resp, err = c.DeleteTopic(&req)
+		return err
```

Comment:
we should still return the (*proto.DeleteTopicsResp, error) pair here, so: nil, err should be the correct way

Reply:
i'm not 100% sure that it's a good idea cause `callOnClusterController` it's a kind of generic function